src/components/colorbar/draw.js

```diff
@@ -10,6 +10,7 @@
 'use strict';
 
 var d3 = require('d3');
+var tc = require('tinycolor2');
```

Comment:
:cow2: we've been using `var tinycolor = require('tinycolor2');` in other files. Let's stay consistent.